dbt/include/athena/macros/utils/timestamps.sql

```diff
@@ -1,5 +1,6 @@
 {%
     pyathena converts time zoned timestamps to strings so lets avoid them now()
+    Iceberg tables and Athena V3 will require a local overwrite of the athena__current_timestamp macro to timestamp(6))
```

Comment:
Considering that athena v3 will become default for everybody can you change this macro to be timestamp(6)? I didn't think that through when I added this haha